tests/director/tests/sensitive_data_strip_test.go

```diff
@@ -0,0 +1,257 @@
+	instanceAuth := fixtures.CreateBundleInstanceAuth(t, ctx, dexGraphQLClient, bndl.ID)
+	require.NotNil(t, instanceAuth)
+
+	t.Run("Application access", func(t *testing.T) {
+		type accessRequired struct {
+			appWebhooks               bool
+			appAuths                  bool
+			bundleInstanceAuth        bool
+			bundleInstanceAuths       bool
+			bundleDefaultInstanceAuth bool
+			documentFetchRequest      bool
+			eventSpecFetchRequest     bool
+			apiSpecFetchRequest       bool
+		}
+		type testCase struct {
+			name              string
+			consumer          *gcli.Client
+			fieldExpectations accessRequired
+		}
+		testCases := []testCase{
+			{
+				name:     "Runtime consumer",
+				consumer: runtimeOAuthGraphQLClient,
+				fieldExpectations: accessRequired{
+					bundleInstanceAuth:        true,
+					bundleInstanceAuths:       true,
+					bundleDefaultInstanceAuth: true,
+				},
+			},
+			{
+				name:              "Integration system consumer",
+				consumer:          intSystemOAuthGraphQLClient,
+				fieldExpectations: accessRequired{},
+			},
+			{
+				name:     "Application consumer",
+				consumer: applicationOAuthGraphQLClient,
+				fieldExpectations: accessRequired{
+					appWebhooks:               true,
+					appAuths:                  true,
+					bundleInstanceAuth:        true,
+					bundleInstanceAuths:       true,
+					bundleDefaultInstanceAuth: true,
+					documentFetchRequest:      true,
+					eventSpecFetchRequest:     true,
+					apiSpecFetchRequest:       true,
+				},
+			},
+			{
+				name:     "Admin user consumer",
+				consumer: dexGraphQLClient,
+				fieldExpectations: accessRequired{
+					appWebhooks:               true,
+					appAuths:                  true,
+					bundleInstanceAuth:        true,
+					bundleInstanceAuths:       true,
+					bundleDefaultInstanceAuth: true,
+					documentFetchRequest:      true,
+					eventSpecFetchRequest:     true,
+					apiSpecFetchRequest:       true,
+				},
+			},
+		}
+
+		for _, test := range testCases {
+			t.Run(test.name, func(t *testing.T) {
+				application := fixtures.GetApplication(t, ctx, test.consumer, tenantId, app.ID)
+				require.Equal(t, application.Webhooks != nil, test.fieldExpectations.appWebhooks)
+				require.Equal(t, application.Auths != nil, test.fieldExpectations.appAuths)
+				require.Equal(t, application.Bundles.Data[0].APIDefinitions.Data[0].Spec.FetchRequest != nil, test.fieldExpectations.apiSpecFetchRequest)
+				require.Equal(t, application.Bundles.Data[0].EventDefinitions.Data[0].Spec.FetchRequest != nil, test.fieldExpectations.eventSpecFetchRequest)
+				require.Equal(t, application.Bundles.Data[0].Documents.Data[0].FetchRequest != nil, test.fieldExpectations.documentFetchRequest)
+				require.Equal(t, application.Bundles.Data[0].InstanceAuths != nil, test.fieldExpectations.bundleInstanceAuths)
+			})
+		}
+	})
+
+	t.Run("Application template access", func(t *testing.T) {
+		t.Run("from runtime", func(t *testing.T) {
```

Comment:
what about accessing app templates from int systems ?

Reply:
the client was the int systems one, just forgot to change the message in th test case